src/vorta/application.py

```diff
@@ -259,7 +264,7 @@ def break_lock(self, profile):
         job = BorgBreakJob(params['cmd'], params)
         self.jobs_manager.add_job(job)
 
-    def bootstrap_profile(self, bootstrap_file=PROFILE_BOOTSTRAP_FILE):
+    def bootstrap_profile(self, bootstrap_file=config.PROFILE_BOOTSTRAP_FILE):
```

Comment:
When is `config.PROFILE_BOOTSTRAP_FILE` evaluated? 

I think it is evaluated when the module is imported. Therefore changes to `config` aren't respected.

Reply:
I'm not sure what to do about this. Could you point me in the right direction?

Reply:
```suggestion
    def bootstrap_profile(self, bootstrap_file=None):
        bootstrap_file = bootstrap_file or config.PROFILE_BOOTSTRAP_FILE
```